scipy/stats/tests/test_distributions.py

```diff
@@ -1064,6 +1064,27 @@ class TestTruncnorm:
     def setup_method(self):
         np.random.seed(1234)
 
+    @pytest.mark.parametrize("a, b, ref",
+    [(1, 1.1, -2.3030441048144877733524473299116314416365215401594087),
+    (0.6, 0.7, -2.3027610681852571124367653023002056177369149733744606),
+    (1, 3, 0.29622333134332169527554642294689301750721849032204733)])
```

Comment:
```suggestion
    [(0, 100, 0.7257913526447274),
    (0.6, 0.7, -2.3027610681852573),
    (1e-6, 2e-6, -13.815510557964274)])
```

Reply:
I added the input values, but I didn't shorten the length of the output, the reason being that I want the outputs to be perfectly reproducible if somebody wants to run the code I used to get them.

Reply:
To make the outputs reproducible you have already provided the actual code that generated them. These type of unit tests compare two double precision values, that's why all throughout the `test_distributions.py` file multiprecision values are converted to doubles first (as far as I see at least). Let us please stay in line with standard scipy testing procedures here. For a big project like scipy, this type of standardization is important even if it might feel annoying sometimes.

Reply:
Fair enough. I've made the changes you requested.